```
src/util.cpp: In function 'void systemCreateDirectory(std::string)':
src/util.cpp:6:31: error: cannot convert 'const char*' to 'LPCWSTR' {aka 'const wchar_t*'}
    6 |     CreateDirectory(path.c_str(), NULL);
      |                     ~~~~~~~~~~^~
      |                               |
      |                               const char*
In file included from C:/msys64/mingw64/x86_64-w64-mingw32/include/winbase.h:18,
                 from C:/msys64/mingw64/x86_64-w64-mingw32/include/windows.h:70,
                 from src/util.hpp:11,
                 from src/util.cpp:1:
C:/msys64/mingw64/x86_64-w64-mingw32/include/fileapi.h:164:55: note:   initializing argument 1 of 'WINBOOL CreateDirectoryW(LPCWSTR, LPSECURITY_ATTRIBUTES)'
  164 |   WINBASEAPI WINBOOL WINAPI CreateDirectoryW (LPCWSTR lpPathName, LPSECURITY_ATTRIBUTES lpSecurityAttributes);
      |                                               ~~~~~~~~^~~~~~~~~~
make: *** [../../compile.mk:72: build/src/util.cpp.o] Error 1
```
```

FIX: 

change

systemCreateDirectory(std::string)

to

systemCreateDirectoryA(std::string)
```

--- a/src/util.cpp
+++ b/src/util.cpp
@@ -3,7 +3,7 @@
 
 void systemCreateDirectory(std::string path) {
 #if ARCH_WIN
-    CreateDirectory(path.c_str(), NULL);
+    CreateDirectoryA(path.c_str(), NULL);
 #else
     mkdir(path.c_str(), 0755);
 #endif

